.buildkite/commands/lint.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash -eu
 
+echo "--- :rubygems: Setting up Gems"
+install_gems
```

Comment:
Seems this command is missing. We'll have to add `plugins: [$CI_TOOLKIT]` to the step calling this `lint.sh` command to fix it 😉

Reply:
Ah yes indeed, just seen CI failing! Added to the pipelines on 4eeff6a0cd47f744877387b31411567155408cd6